docs/users_guide/supported_datasets.rst

```diff
@@ -1,104 +1,146 @@
+     - Yes
+     - Needs testing
+     - Needs testing
+   * - `CESM/CAM-chem SE <https://www2.acom.ucar.edu/gcm/cam-chem>`_
+     - Yes
+     - | Needs testing & to 
+       | add unstructured 
+       | grid capabilities
+     - | Needs testing & to 
+       | add unstructured 
+       | grid capabilities
+   * - `CMAQ <https://www.epa.gov/cmaq/>`_
+     - Yes
+     - Needs testing
+     - Needs testing
+   * - `UFS-AQM (RRFS) <https://github.com/ufs-community/ufs-srweather-app/wiki/Air-Quality-Modeling>`_
+     - Yes
+     - Yes
+     - Needs testing
+   * - `CAMx <https://www.camx.com/>`_
+     - Yes
+     - Needs testing
+     - TROPOMI, TEMPO
+   * - `RAQMS <http://raqms-ops.ssec.wisc.edu/>`_
+     - Yes
+     - Needs testing
+     - MOPITT, OMPS
+
+In general, processing requires input to be in netCDF format. For the above 
+models, scripts to configure the model data into a standard format for 
+MELODIES MONET are available. If input datasets are in netCDF format and  
+define latitude, longitude, pressure, and a datetime object, MELODIES MONET may be able 
+to directly read the input files.
+
+Please refer to the
+`MELODIES MONET project board <https://github.com/orgs/NOAA-CSL/projects/6>`__ 
+to learn more about our current and future development plans.
+
+Supported Observations
+----------------------
+
+Surface
+^^^^^^^
+To use these surface datasets in MELODIES MONET specify "obs_type" equal to "pt_sfc" in your YAML file. Currently, 
+surface data in MELODIES MONET is paired horizontally using the nearest neighbor approach and temporally with 
+exact time matching (e.g., if model data is provided hourly and observational data is provided 
+hourly, these will pair together). More time interpolation options will be added in the future.
+
+Available now:
+   * `AirNow <https://www.airnow.gov/>`_ - Air quality data aggregated by AirNow
+   * `AERONET <https://aeronet.gsfc.nasa.gov/>`_ - NASA AErosol RObotic NETwork
+
+   * `AQS <https://www.epa.gov/aqs/>`_ - US EPA Air Quality System
+   * `ISH and ISH-Lite <https://www.ncei.noaa.gov/products/land-based-station/integrated-surface-database>`_ - NOAA National Centers for Environmental Information (NCEI) Global Hourly - Integrated Surface Database
+   * `OpenAQ <https://openaq.org/>`_ - Global, aggregated, harmonized public air quality data hosted by OpenAQ
+
+Under Development:
+   * `IMPROVE <http://vista.cira.colostate.edu/Improve/>`_ - Interagency Monitoring of Protected Visual Environments
+   * `CRN <https://www.ncdc.noaa.gov/crn/>`_  - U.S. Climate Reference Network 
+   * `TOLNet <https://www-air.larc.nasa.gov/missions/TOLNet/>`_ - Tropospheric Ozone Lidar Network
+   * `CEMS <https://www.epa.gov/emc/emc-continuous-emission-monitoring-systems/>`_ - Air Emission Measurement Center (EMC) Continuous Emission Monitoring System
+   * `Pandora <https://pandora.gsfc.nasa.gov/>`_ - Part of the `Pandonia Global Network <https://www.pandonia-global-network.org/>`_

```

Comment:
I believe Pandonia Global Network (PGN) is only Pandora instruments, nothing else. The PGN project provides the data.
```suggestion
   * `Pandora <https://pandora.gsfc.nasa.gov/>`_ (`Pandonia Global Network <https://www.pandonia-global-network.org/>`_)
```

Reply:
I just took it from their website so it appears that the NASA Pandora network across the US is part of the Pandonia Global Network, but the parentheses also works fine to convey this.